onnxscript/ir/_tape.py

```diff
@@ -85,17 +85,23 @@ def op(
         name: str | None = None,
         doc_string: str | None = None,
         metadata_props: dict[str, str] | None = None,
+        output: ir.Value | None = None,
     ) -> ir.Value:
         if attributes is None:
             attrs: Sequence[ir.Attr | ir.RefAttr] = ()
         else:
             attrs = _convenience.convert_attributes(attributes)
+        output_kwargs: dict[str, Any]
```

Comment:
op and op_multi_output are very similar. Maybe we could merge them into a single method.